packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -120,8 +120,13 @@ export default class DashboardStore implements IDashboardStore {
     strategy_save_type = 'unsaved';
     active_tour_step_number = 0;
     show_toast = false;
+    toast_message = '';
     is_file_supported = false;
 
+    setToastMessage = (toast_message: string) => {
```

Comment:
@farabi-deriv look at the correct example func setShowToast(), check it how it has done in the store dashboard through all files. setShowToast() should :
1.  add type in the interface IDashboardStore above
2. if you use this method not only in the store mobx , but also pass it to components, then it should  be in the list of arguments which is an object in the` makeObservable()`

Reply:
Yes, correct. Missed out on adding there. Thanks. This is done.